claims-related/tests/single_source_csv/csv_report/Measure014.md

```diff
@@ -0,0 +1,13 @@
+# Comparison for Measure ID 14
+Data Element Name Summary
+*Added DEN*:set()
+*Removed DEN*:set()
+*Changed DEN*:{'ENCOUNTER_CODE'}
+## For Data Element Name ENCOUNTER_CODE
+Codes Summary
+* Number of Added CODES*:0
+* Number of Removed CODES*:1
+* Number of Changed CODES*:0
+### Removed Codes
+VERSION Measure_ID DATA_ELEMENT_NAME CODING_SYSTEM  CODE    MODIFIER PLACE_OF_SERVICE AGE GENDER
+   BASE         14    ENCOUNTER_CODE            C4 99201 ≠GQ, GT, 95              ≠02 ≥50   M, F
```

Comment:
This doesn't actually display as a table in markdown; it's just one long line.  Getting an actual markdown table might be more trouble than it's worth, but it would be nice to do a list of key: value pairs on new lines.

VERSION: BASE
Measure_ID: 14
...